ci-operator/step-registry/telcov10n/functional/hcp-cnf/tests/telcov10n-functional-hcp-cnf-tests-ref.yaml

```diff
@@ -0,0 +1,24 @@
+ref:
+  as: telcov10n-functional-hcp-cnf-tests
+  from_image:
+    namespace: ci
+    name: telco-runner
+    tag: latest
+  commands: telcov10n-functional-hcp-cnf-tests-commands.sh
+  timeout: 4h15m
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: T5CI_VERSION
+    default: ""
+    documentation: Version of OCP cluster to install.
+  - name: T5CI_JOB_TYPE
+    default: ""
+    documentation: Job type of Telco5G CI jobs
+  - name: TEST_RUN_FEATURES
```

Comment:
I didn't see this parameter in the config, are you using it?

Reply:
For now it is not used but will be used probably inside the tests, I'll remove it though.